### PR TITLE
add: forgot password

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add new page and new route
- Add texts
- TODO add final change language styles.

image

### CODE REVIEW

**Comment by andresmoschini** [comment]
Remember adding the links to this page.

File: src/App.js [hunk 1]
```diff
@@ -79,6 +80,7 @@ class App extends Component {
             />
             <Route path="/login/" exact component={Login} />
             <Route path="/signup/" exact component={Signup} />
+            <Route path="/forgotpassword/" exact component={ForgotPassword} />
```
Comment (andresmoschini) [inline src/App.js]: What do you think about

```suggestion
            <Route path="/forgot-password/" exact component={ForgotPassword} />
```

File: src/components/ForgotPassword/ForgotPassword.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    await timeout(1500);
+    setSubmitting(false);
+  };
+
+  return (
+    <main className="panel-wrapper">
+      <article className="main-panel">
+        <header>
+          <h1 className="logo-doppler-new">Doppler</h1>
+          <div className="language-selector">
```
Comment (andresmoschini) [inline src/components/ForgotPassword/ForgotPassword.js]: Maybe in another PR, but, what do you think about moving this code into a component? (We already meet [the rule of three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)))

File: src/components/ForgotPassword/ForgotPassword.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+const ForgotPassword = ({ intl }) => {
+
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const validate = (values) => {
+    const errors = {};
+
+    if (!values[fieldNames.email]) {
+      errors[fieldNames.email] = _('validation_messages.error_required_field');
+    } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values[fieldNames.email])) {
```
Comment (cbernat) [inline src/components/ForgotPassword/ForgotPassword.js]: Maybe add this regex in a more reusable place? you have password validation in signup and here.

Comment (andresmoschini) [reply src/components/ForgotPassword/ForgotPassword.js]: You are right, although this validation is still pending, there is a task to deal with these things: [DBR-135](http://jira.makingsense.com/browse/DBR-135)

Comment (fcoronelmakingsense) [reply src/components/ForgotPassword/ForgotPassword.js]: Yes, but there is another task for validations

File: src/components/ForgotPassword/ForgotPassword.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/components/ForgotPassword/ForgotPassword.js]: Yes i think it's better

**Comment by andresmoschini** [comment]
> TODO add final change language styles.

If you are being blocked for this, do not worry, follow the same approach that you followed in the other pages, and then, @damianmuti could fix the HTML along with https://github.com/FromDoppler/Doppler-UI-System/pull/81

File: src/App.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/App.js]: changed

**Comment by fcoronelmakingsense** [comment]
TODO @damianmuti apply language selector

**Pull request merged**